docs/integration/stage_files.md

```diff
@@ -0,0 +1,98 @@
+# Stage Your Files on Request
+
+After you've [designated a file staging area and configured a Globus guest
+collection](staging_area.md) for your organization, it's time to set up a
+_staging request endpoint_ that allows the Data Transfer System (DTS) to ask for
+specific files to be moved to this staging area in preparation for transfer.
+
+Why is this endpoint needed? In a perfect world, storage would be infinite and
+free, and your organization could simply "keep all the files somewhere, ready to
+be transferred anytime." In practice, the amount of scientific data being
+continuously generated greatly exceeds available storage at any given moment,
+so it needs to be archived when it's not being used.
+
+For example, the JGI Data Portal (JDP) stores most of its data on tape,
+unarchiving it to disk (the staging area used by the DTS, specifically) on
+request. Your organization probably has a similar scheme in place for storing
+its data.
+
+It works like this: when a user searches for and selects files to transfer from
+your database to somewhere else, the DTS checks your staging area for the
+requested files via Globus. If those files are not in your staging area, the DTS
+makes a request to the staging request endpoint to ask your system to copy the
+files into place, whatever that entails.
+
+The DTS understands that this process may take some time, so it also needs a way
+to request the status of your file staging operation. After all, there are only
+so many robot arms in a tape backup facility, and they can only swing around so
+quickly and move so many tapes at once. Accordingly, your system must provide
```

Comment:
what? I thought it was the data equivalent of Oompa Loompas!

Reply:
I suppose equipment and approaches vary from site to site!